packages/arb-token-bridge-ui/src/tokenLists.tsx

```diff
@@ -43,7 +43,7 @@ export const BRIDGE_TOKEN_LISTS: BridgeTokenList[] = [
     id: 4,
     originChainID: '421611',
     url: 'token-list-421611.json',
-    name: 'Rinkarby Tokens',
+    name: 'ArbRinkeby',
```

Comment:
Any reason for the changes? Did you mean to change the URL?

Reply:
no, this one is for the array filter matching
> BRIDGE_TOKEN_LISTS.find(list => list.name.includes(data.name))

Reply:
oh sorry let me remove the previous commit